cypress/e2e/features/backend/usersubscription.feature

```diff
@@ -0,0 +1,11 @@
+Feature: Burak
+
+    Background: User gets idToken
+    Given User gets idToken
+
```

Comment:
There is already function for generating token [(),](https://github.com/abeyis/Revayz/blob/cade1c9087ba8f695527061cec66a707c6d98265/cypress/e2e/features/backend/aa_token_generate.feature#L3) instead of recreating function you can use that function. You may need to import generated token from that file, but it is good to use existing function, instead of creating a new function.